runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -137,6 +137,7 @@ def commonExcludeCategories = [
   'org.apache.beam.sdk.testing.UsesGaugeMetrics',
   'org.apache.beam.sdk.testing.UsesSetState',
   'org.apache.beam.sdk.testing.UsesMapState',
+  'org.apache.beam.sdk.testing.UsesTimerMap',
```

Comment:
do we need to exclude for Dataflow?

Reply:
If it was broken, we need to fix. Support for this was added for Dataflow, though it now appears that the test was never being run :(

Reply:
Yes, My first approach was to enable for Dataflow but then the build fail on the `UsesTimerMap` tests:
https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_PR/210/
https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_PortabilityApi_Dataflow_PR/76/